src/js/helpers/cleanData.js

```diff
@@ -283,7 +283,73 @@ export const cleanDepartmentDirectors = directors => {
   });
 };
 
-export const cleanDepartments = allDepartments => {
+export const cleanDepartmentTopServiceLink = (topService, langCode) => {
```

Comment:
I haven't tested this yet and it looks solid, but it feels like a lot of work for the frontend to have to do just for making URLs.

Reply:
it is less than ideal, but it isn't far removed from our existing logic around making links work. There's a bit extra in here to make sure we retain the page IDs so we can avoid having any top services show up as related information.

Since related information links are populated by looking through every page and finding parent departments during the frontend build process, as opposed to being managed in Joplin, this seemed like the best place for the logic.

This type of thing is a great example of why https://github.com/cityofaustin/techstack/issues/2792 exists